beets/ui/__init__.py

```diff
@@ -1034,7 +1042,7 @@ def parse_global_options(self, args):
         """Parse options up to the subcommand argument. Returns a tuple
         of the options object and the remaining arguments.
         """
-        options, subargs = self.parse_args(args)
+        options, subargs = self.parse_args(_convert_args(args))
```

Comment:
because optparse only accepts strings, so it first breaks when seeing `--foo` type options